From 9cd30967637785292b4adc758f66dc14c59b9f79 Mon Sep 17 00:00:00 2001
From: Guilhem IMBERTON <guilhem.imberton@intel.com>
Date: Wed, 28 Nov 2012 17:00:28 +0100
Subject: [PATCH 59/78] BUILD: Allow customers to use their own product/device
 name

BZ: 70731

Use dedicated variable instead of TARGET_PRODUCT/TARGET_DEVICE
where needed.
Remove product name from kernel/module build scripts.
All out-of-tree kernel modules use same build script.

Change-Id: I5a2d9da19435f05678a643a5ed25b87099dd5ab3
Signed-off-by: Guilhem IMBERTON <guilhem.imberton@intel.com>
Reviewed-on: http://android.intel.com:8080/79985
---
 wl12xx-compat/AndroidWl12xxCompat.mk |    6 +++++-
 wl12xx-compat/Makefile               |    1 -
 wlan_prov/Android.mk                 |    2 +-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/wl12xx-compat/AndroidWl12xxCompat.mk b/wl12xx-compat/AndroidWl12xxCompat.mk
index b9fea9b..6e70fc9 100644
--- a/wl12xx-compat/AndroidWl12xxCompat.mk
+++ b/wl12xx-compat/AndroidWl12xxCompat.mk
@@ -2,6 +2,10 @@
 
 .PHONY: build_wl12xx-compat
 build_wl12xx-compat: build_kernel
-	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(TARGET_DEVICE)
+	+TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" \
+	TARGET_DEVICE="$(TARGET_DEVICE)" \
+	TARGET_BOARD_PLATFORM="$(TARGET_BOARD_PLATFORM)" \
+	vendor/intel/support/kernel-build.sh \
+	-X hardware/ti/wlan/wl12xx-compat
 
 $(PRODUCT_OUT)/ramdisk.img : build_wl12xx-compat
diff --git a/wl12xx-compat/Makefile b/wl12xx-compat/Makefile
index 8863292..4a90f4d 100644
--- a/wl12xx-compat/Makefile
+++ b/wl12xx-compat/Makefile
@@ -34,7 +34,6 @@ endif
 endif
 
 else
-
 export PWD :=	$(shell pwd)
 CFLAGS += \
         -DCOMPAT_BASE_TREE="\"$(shell cat compat_base_tree)\"" \
diff --git a/wlan_prov/Android.mk b/wlan_prov/Android.mk
index 79f7695..c123fa6 100644
--- a/wlan_prov/Android.mk
+++ b/wlan_prov/Android.mk
@@ -17,7 +17,7 @@ LOCAL_STATIC_LIBRARIES := \
 	CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB
 endif
 
-ifeq ($(TARGET_PRODUCT),mfld_gi)
+ifeq ($(REF_DEVICE_NAME),mfld_gi)
 LOCAL_CFLAGS += -DSINGLE_BAND
 endif
 
-- 
1.7.9

